src/Microsoft.VisualStudio.Editors/PropPages/DebugPropPage.resx

```diff
@@ -984,6 +984,12 @@
   <metadata name="$this.Localizable" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
     <value>True</value>
   </metadata>
+  <data name="$this.AutoScaleDimensions" type="System.Drawing.SizeF, System.Drawing">
+    <value>6, 13</value>
```

Comment:
This is the scaling factor, see: https://msdn.microsoft.com/en-us/library/system.windows.forms.containercontrol.autoscaledimensions(v=vs.110).aspx for more information.

Anything that is:

1) Using the default designer Font (Tahoma, or MS Sans Serif) 8.25 PT
2) Designed at 96 DPI

Must be 6, 13. If they are not, then they are incorrect and will be scaling incorrectly.